frontend/src/app/modules/fields/edit/field-types/project-status-edit-field.component.ts

```diff
@@ -31,15 +31,15 @@ import {Component, OnInit, ViewChild} from "@angular/core";
 import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {projectStatusCodeCssClass, projectStatusI18n} from "core-app/modules/fields/helpers/project-status-helper";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3b1eae5a64159f8a044579a25870c772186dbd3f/frontend/src/app/modules/fields/edit/field-types/project-status-edit-field.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>